dotCMS/src/integration-test/java/com/dotcms/rendering/velocity/services/HTMLPageAssetRenderedTest.java

```diff
@@ -968,6 +1002,14 @@ public void shouldReturnPageHTMLForPersona() throws Exception{
         Assert.assertEquals(html , "content2content1");
     }
 
+    private HttpSession createHttpSession(HttpServletRequest mockRequest) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'mockRequest' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5067852)